src/pages/studies/StudyTableContainer.tsx

```diff
@@ -23,11 +23,20 @@ const mapState = (state: RootState): StudyTableContainerState => ({
 const connector = connect(mapState, mapDispatch);
```

Comment:
I would change the name of the file (remove "Container")

Reply:
this comment would have be more useful 1 month ago when it was merge, but anyway, I'll chage it